salt/orch/kubernetes.sls

```diff
@@ -19,12 +19,20 @@ update_pillar:
     - require:
       - salt: hostname_setup
 
+update_grains:
+  salt.function:
+    - tgt: '*'
+    - name: saltutil.refresh_grains
```

Comment:
We do, we're now on 2016.11.4 which supports it.

Is it needed for update-etc-hosts.sls? I'll add it, it's should be harmless + safer anyway